src/pages/en/guides/deploy/aws.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Deploy your Astro Site to AWS
+description: How to deploy your Astro site to the web using AWS.
+layout: ~/layouts/DeployGuideLayout.astro
+---
+
+[AWS](https://cloud.google.com/) is a full-featured web app hosting platform that can be used to deploy an Astro site.
+
+## How to deploy
+
+Deploying your project to AWS requires using the [AWS console](https://aws.amazon.com/console/). (Most of these actions can also be done using the [AWS CLI](https://aws.amazon.com/cli/)). This guide will walk you through the steps to deploy your site to AWS from the most basic method then adding more services on top to improve cost efficiency and performance.
+
+### S3 Bucket
+
+S3 is the starting point of any application. It is where your proyect files and other assets are stored. S3 charges for file storage and number requests. You can find more information about S3 in the [AWS documentation](https://aws.amazon.com/s3/).
+
+1. Create an S3 bucket with your project's name.
+
+:::note
+The bucket name should be globally unique. We recommend combination of your project name and the domain name of your site.
+:::
+
+2. Disable "Block all public access". By default, AWS sets all buckets to be private. To make it public, you need to unchecked the "Block public access" checkbox in the bucket's properties.
+
+3. Upload your built files located in `dist` to S3. You can do this manually in the console or use the AWS CLI. If you use the AWS CLI, you can use the following command. 
+
+  ```
+  aws s3 cp dist/ s3://<BUCKET_NAME>/ --recursive
+  ```
+
+:::note
+Do not forget to replace `<BUCKET_NAME>` with the name of your bucket and to authenticate your CLI with your AWS credentials.
+:::
+
+4. Update your bucket policy to allow public access. You can find this setting in bucket's Permissions > Bucket policy.
```

Comment:
I’d add a “the” here. And what do you think about bolding the Amazon UI elements?

```suggestion
4. Update your bucket policy to allow public access. You can find this setting in the bucket's **Permissions > Bucket policy**.
```

Reply:
We’ve also used `<kbd>` elements in some places, e.g. https://docs.astro.build/en/guides/deploy/netlify/#website-ui-deployment so maybe that could make sense if you click first <kbd>Permissions</kbd>, then <kbd>Bucket Policy</kbd>?

```suggestion
4. Update your bucket policy to allow public access. You can find this setting in the bucket's <kbd>Permissions</kbd> > <kbd>Bucket policy</kbd>.
```